src/feature/flow_chart_panel/components/node-edit-menu/ParamField.tsx

```diff
@@ -60,6 +62,10 @@ const ParamField = ({
       );
     case "select":
       return <Select onChange={handleChange} data={options!} value={value} />;
+    case "node_reference":
+      return (
+        <Select onChange={handleChange} data={otherNodeIds!} value={value} />
```

Comment:
Just showing ID's can be confusing as all node id's start with node function name so for duplicate nodes it'll show same name. We can display node labels for better UX.
e.g:
![Screenshot (40)](https://github.com/flojoy-io/studio/assets/86223939/97b870a7-c6a3-423a-a625-512f5b7d317a)